### DISCUSSION TITLE
A Secret Of The Debug Class

### DISCUSSION THREAD

**Discussion by partisani** [CosmosOS/Cosmos#1972] [Q&A]
I've Find A Other Class Inside Of Debug Class

`
Cosmos.Debug.Kernel.Debugger.SendKernelPanic(id: 0);
`

But I Don't Know How To Use. Can Anyone Help Me?

**Comment by quajak** [comment]
SendKernelPanic will cause Cosmos to terminate and show the last address and output the kernel panic you threw.

**Reply by partisani** [reply]
I Have Tried It But Nothing Happened

**Reply by quajak** [reply]
What VM are you using? On Bochs it works for sure, havnt tested it on VMWare though.
